examples/resources/mondoo_integration_crowdstrike/resource.tf

```diff
@@ -0,0 +1,22 @@
+variable "client_id" {
+  description = "The Client ID"
+  type        = string
+  sensitive   = true
+}
+
+variable "client_secret" {
```

Comment:
I think we should add the cloud into the example since its most likely users are not on the standard cloud